src/lib.rs

```diff
@@ -71,7 +72,11 @@ impl AttributeData {
 /// General structure that contains points and attached feature attributes.
 pub struct PointData {
     pub position: Vec<Vector3<f64>>,
-    pub attributes: FnvHashMap<String, AttributeData>,
+    // BTreeMap for deterministic iteration order.
+    pub attributes: BTreeMap<String, AttributeData>,
 }
 
 pub use point_viewer_proto_rust::proto;
+
+/// size for batch
+pub const NUM_POINTS_PER_BATCH: usize = 500_000;
```

Comment:
Nit: as this refers to the size of PointData and not anymore to the Batch Iterator, could the name reflect more this change

Reply:
@nnmm @catevita Or we even rename `PointData` to `PointBatch`? What do you think?

Reply:
+1

Reply:
Done, I renamed it to `PointsBatch`